spec/fixtures/coffee.coffee

```diff
@@ -13,6 +13,7 @@ class quicksort
       if current < pivot
         left.push(current)
       else
+        # coffeelint: disable=no_trailing_semicolons
```

Comment:
That makes sense - we could handle that by adding a `.coffeelintignore` (oh how I wish we could just configure it in `coffeelint.json`), but that would be redundant with the coffeelint grunt task config. So I could just revert the changes to fixtures, if that would work?